connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/resources/ConnectorsResource.java

```diff
@@ -97,6 +100,15 @@ public ConnectorsResource(Herder herder, WorkerConfig config) {
         isTopicTrackingResetDisabled = !config.getBoolean(TOPIC_TRACKING_ALLOW_RESET_CONFIG);
     }
 
+    // For testing purposes only
+    public static void setRequestTimeout(long requestTimeoutMs) {
```

Comment:
Can this be knocked down to protected / package-private?

Reply:
Don't think so :(
Right now it's really only useful to call this from an integration test, and those all live in a different package. We could move to a different package but it seems a little funky to put one of our integration tests in a different place from all the others, or to move all of our integration tests into the seemingly-arbitrary `org.apache.kafka.connect.runtime.rest.resources` package.